.claude/skills/llm-json-schema-documentation.md

```
@@ -0,0 +1,184 @@
+
+### 3. Show Examples, Not Templates
+
+**✅ Examples** (Reference - show good patterns):
+```markdown
+**Example Output:**
+```markdown
+[SOCIAL SKILL CHALLENGE: Lady Ashwood]
+Objective: Access family archives
+Social HP: 5/5 | Status: RESISTING
+```
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)